translations/README.md

```diff
@@ -0,0 +1,24 @@
+## Translation guidelines
+
+### General guidelines
+
+When possible, please use the following guidelines when translating field names:
+
+- Entity type names should **not** be included in the label, when it is redundant
+  - _Example_: `First name` instead of `User first name`
+- Field names should have only their first word capitalized, per FOLIO UI guidelines
+  - _Example_: `First name`, `Effective call number`
+- Identifier acronyms such as ID, HRID, etc., should always be fully capitalized
+- As long as it is not ambiguous, use 'ID' instead of 'UUID'
+- Avoid using abbreviations/acronyms, unless they are used universally within their application
+  - _Example_: `HRID` in lieu of `Human readable identifier` because `HRID` is used universally within the inventory applications, and `Human readable identifier` is never used
+  - _Example_: `ISBN` in lieu of `International standard book number`, for the same reason
+  - _Example_: **avoid** using `Acq. unit` in lieu of `Acquisition unit`
+  - _Example_: **avoid** using `POL` instead of `Purchase order line`
```

Comment:
How do we feel about this? This also may not matter anymore, with the first point

Reply:
I think this makes sense. It's better than having some things be abbreviated and some not. The only thing I'm not sure about is POLs, I was thinking someone specifically requested that it be POL at one point.

Reply:
Yeah, that's the one I'm iffiest on — it's an "industry/common-knowledge term" I guess?